Ruby/lib/algorithms.rb

```diff
@@ -4,17 +4,41 @@
 # The result is called a 'digital root'.
 # Do not use string conversion within your method.
 def digital_root(number)
-
+  # sum = 0
+  number = number.to_s.split('').map(&:to_i)
```

Comment:
This works and is a good solution, but actually violates one of the instructions: "Do not use string conversion within your method." The specs pass though, so maybe I should add an expectation that fails when `.to_s` is called.